web/src/main/scala/net/wiringbits/AppRouter.scala

```diff
@@ -44,6 +44,7 @@ import scala.util.{Failure, Success}
     val emailCode = route("/verify-email/:emailCode", props.ctx)(VerifyEmailWithTokenPage(props.ctx))
     val forgotPassword = route("/forgot-password", props.ctx)(ForgotPasswordPage(props.ctx))
     val resetPassword = route("/reset-password/:resetPasswordCode", props.ctx)(ResetPasswordPage(props.ctx))
+    val resendVerifyEmail = route("/resend-verify-email", props.ctx)(ResendVerifyEmailPage(props.ctx))
```

Comment:
Do we really need a new view for this?

Reply:
Yes, because without it we can't render the resend verify email page

Reply:
I just saw the screenshot, it is clear now, we have a separate view to re-send the verification email.